mlx/backend/cuda/quantized/quantized_utils.cuh

```diff
@@ -15,6 +15,26 @@ inline constexpr __device__ short get_bytes_per_pack() {
   return power_of_2_bits ? (wsize / 8) : (bits == 5 ? 5 : 3);
 }
 
+template <typename T>
+__device__ __forceinline__ void abs_max_x2(T& out, const T& x1, const T& x2) {
+  if constexpr (std::is_same<T, __nv_bfloat162>::value) {
+    asm volatile("max.xorsign.abs.bf16x2 %0, %1, %2;\n"
+                 : "=r"(reinterpret_cast<uint32_t&>(out))
+                 : "r"(reinterpret_cast<const uint32_t&>(x1)),
+                   "r"(reinterpret_cast<const uint32_t&>(x2)));
```

Comment:
Does `__habs2` and `__hmax2` work here? Could be a bit simpler than the inline assembly.